packages/inspection-capture-web/src/PhotoCapture/hooks/usePictureTaken.ts

```diff
@@ -44,19 +44,32 @@ export function usePictureTaken({
   tasksBySight,
 }: UseTakePictureParams): HandleTakePictureFunction {
   const { trackEvent } = useAnalytics();
-  return useCallback(
+  return useCallback<HandleTakePictureFunction>(
     (picture: MonkPicture) => {
       sightState.setLastPictureTakenUri(picture.uri);
-      const upload: PictureUpload =
-        addDamageHandle.mode === PhotoCaptureMode.SIGHT
-          ? {
-              mode: addDamageHandle.mode,
-              picture,
-              sightId: sightState.selectedSight.id,
-              tasks: tasksBySight?.[sightState.selectedSight.id] ?? sightState.selectedSight.tasks,
-            }
-          : { mode: addDamageHandle.mode, picture };
-      uploadQueue.push(upload);
+      switch (addDamageHandle.mode) {
+        case PhotoCaptureMode.ADD_DAMAGE_1ST_SHOT:
+        case PhotoCaptureMode.ADD_DAMAGE_2ND_SHOT:
+          uploadQueue.push({ mode: addDamageHandle.mode, picture });
+          break;
+        case PhotoCaptureMode.SIGHT:
+          uploadQueue.push({
+            mode: addDamageHandle.mode,
+            picture,
+            sightId: sightState.selectedSight.id,
+            tasks: tasksBySight?.[sightState.selectedSight.id] ?? sightState.selectedSight.tasks,
+          });
+          break;
+        case PhotoCaptureMode.ADD_DAMAGE_PART_SELECT:
```

Comment:
move this switch case above the `SIGHT` case to make it next to the other add damage cases